nixos/modules/system/boot/systemd.nix

```diff
@@ -1,7 +1,7 @@
 { config, lib, pkgs, utils, ... }:
 
-with lib;
 with utils;
+with lib;
```

Comment:
If the order is that important, perhaps a comment would be warranted. Is this picking `lib` from `utils`?